okapi-core/src/main/raml/ModuleDescriptor.json

```diff
@@ -80,6 +80,10 @@
     "launchDescriptor": {
       "description": "Default deployment for this module",
       "$ref": "LaunchDescriptor.json"
+    },
+    "user" : {
```

Comment:
Do we foresee the need to ever define more than one module user?

Reply:
How would those be distinguished from one another? This is per module it seems.

Reply:
I also suggest multiple module users with different permissions. For example a migration might require more permissions than the regular work.
We can easily add some identifier to distinguish them.

Reply:
The additional deployment complexity of managing identifiers for users and providing of multiples usernames along with credentials to the module environment variables without corresponding business requirements seems to be a premature optimization. It is also inconvenient to provide Map<String,String> of arbitrary size (username and pass for each user) in environment variable.